synapse/lib/agenda.py

```diff
@@ -856,7 +856,7 @@ async def _runJob(self, user, appt):
             starttime = self._getNowTick()
             success = False
             try:
-                opts = {'user': user.iden, 'view': appt.view}
+                opts = {'user': user.iden, 'view': appt.view, 'vars': {'_iden': appt.iden}}
```

Comment:
Is there a chance in the future we would want to stuff more info in, and then maybe it makes sense to take the hit on one  var (like a $_self) that's a dict?

Reply:
Maybe? We already have the `$tag` and `$prop` being mapped in on triggers though, so we'd end up with either duplicates or a schism, but i could def see `$auto.iden` maybe being useful as the "unified way to represent the iden of an automation element"